app/(gcforms)/[locale]/(form administration)/form-builder/components/dialogs/DynamicRowDialog/DynamicRowDialog.tsx

```diff
@@ -0,0 +1,264 @@
+              <h4 className="mb-4 block font-bold">{t("dynamicRow.dialog.addButton.title")}</h4>
+              <p className="mb-4 text-sm">{t("dynamicRow.dialog.addButton.description")}</p>
+              <TextInput label={t("dynamicRow.dialog.english")}>
+                <input
+                  aria-label={addButtonTextA11yEn}
+                  value={item.properties.dynamicRow?.addButtonTextEn}
+                  onChange={(e) => {
+                    setItem({
+                      ...item,
+                      properties: {
+                        ...item.properties,
+                        dynamicRow: {
+                          ...(item.properties.dynamicRow as dynamicRowType),
+                          addButtonTextEn: e.target.value || "",
+                        },
+                      },
+                    });
+                  }}
+                />
+              </TextInput>
+              <TextInput label={t("dynamicRow.dialog.french")}>
+                <input
+                  aria-label={addButtonTextA11yFr}
+                  value={item.properties.dynamicRow?.addButtonTextFr}
+                  onChange={(e) => {
+                    setItem({
+                      ...item,
+                      properties: {
+                        ...item.properties,
+                        dynamicRow: {
+                          ...(item.properties.dynamicRow as dynamicRowType),
+                          addButtonTextFr: e.target.value || "",
+                        },
+                      },
+                    });
+                  }}
+                />
+              </TextInput>
+            </div>
+
+            {/* Remove button input */}
+            <div>
+              <label className="mb-4 block font-bold">
+                {t("dynamicRow.dialog.removeButton.title")}
+              </label>
+              <p className="mb-4 text-sm">{t("dynamicRow.dialog.removeButton.description")}</p>
+              <TextInput label={t("dynamicRow.dialog.english")}>
+                <input
+                  aria-label={removeButtonTextA11yEn}
+                  value={item.properties.dynamicRow?.removeButtonTextEn}
+                  onChange={(e) => {
+                    setItem({
+                      ...item,
+                      properties: {
+                        ...item.properties,
+                        dynamicRow: {
+                          ...(item.properties.dynamicRow as dynamicRowType),
+                          removeButtonTextEn: e.target.value || "",
+                        },
+                      },
+                    });
+                  }}
+                />
+              </TextInput>
+              <TextInput label={t("dynamicRow.dialog.french")}>
+                <input
+                  aria-label={removeButtonTextA11yFr}
+                  value={item.properties.dynamicRow?.removeButtonTextFr}
+                  onChange={(e) => {
+                    setItem({
```

Comment:
Wondering if we should make a small util to avoid all the duplicated spreads etc... across the diff updates

Really just need the single prop being updated and the rest is the same.

Reply:
https://github.com/cds-snc/platform-forms-client/pull/4446/commits/7c9470c10b3fd895d480edbc86ecb18dac8e5ff9